src/pages/FormPage.tsx

```diff
@@ -0,0 +1,172 @@
+import { FormEvent, useRef, useState } from "react";
+import "./FormPage.scss";
+
+const FormPage = () => {
+  const [submitted, setSubmitted] = useState(false);
+  const form = useRef<HTMLFormElement>(null);
+
+  const handleClick = () => {
+    form.current?.requestSubmit();
+  };
+
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
+    if (form.current) {
+      const formData = new FormData(form.current);
+      formData.forEach((value, key) => {
+        console.log("Key:", key, "Value:", value);
+      });
+    }
+    setSubmitted(true)
+    setTimeout(() => {
+        document.querySelector('tds-toast')?.hideToast()
+    }, 5000)
+  };
+
+  return (
+    <>
+    <article>
+      <form
+        ref={form}
+        onSubmit={(event: FormEvent) => {
+          handleSubmit(event);
+        }}
+      >
+        <h3>Form example</h3>
+        <section>
+          <h4>Text Input</h4>
+          <tds-text-field
+            name="text-field"
+            label="Full name"
+            label-position="outside"
+            placeholder="John Doe"
+          ></tds-text-field>
+          <tds-datetime
+            label="Date of birth"
+            name="dateOfBirth"
+            type="date"
+            default-value="2000-01-01"
+          ></tds-datetime>
+        </section>
+        <section>
+          <h5>Which offices do you work from?</h5>
+          <tds-checkbox value="sodertalje321" name="office">
+            <div slot="label">Södertälje 321</div>
+          </tds-checkbox>
+          <tds-checkbox value="sergel" name="office">
+            <div slot="label">Sergel</div>
+          </tds-checkbox>
+          <tds-checkbox value="homeoffice" name="office">
+            <div slot="label">Home office</div>
+          </tds-checkbox>
+          <h5>How long have you been at Scania?</h5>
+          <tds-radio-button checked value="radio-1" name="yearsAtScania">
+            <div slot="label">0 - 6 months</div>
+          </tds-radio-button>
+          <tds-radio-button value="radio-2" name="yearsAtScania">
+            <div slot="label">6 months - 1 year</div>
+          </tds-radio-button>
+          <tds-radio-button value="radio-2" name="yearsAtScania">
+            <div slot="label">1 - 2 years</div>
+          </tds-radio-button>
+          <tds-radio-button value="radio-2" name="yearsAtScania">
+            <div slot="label">2 - 5 years</div>
+          </tds-radio-button>
+          <tds-radio-button value="radio-2" name="yearsAtScania">
+            <div slot="label">+ 5 years</div>
+          </tds-radio-button>
+        </section>
+        <tds-textarea
+          name="textarea"
+          label="What do you do at Scania?"
+          label-position="outside"
+        ></tds-textarea>
```

Comment:
To add some spacing between, just cosmetic stuff...
```suggestion
   <section className="tds-u-pt3">
          <tds-textarea
              name="textarea"
              label="What do you do at Scania?"
              label-position="outside"
          ></tds-textarea>
        </section>
```